src/org/javarosa/xpath/expr/XPathExpression.java

```diff
@@ -76,9 +76,10 @@ public Object pivot (DataInstance model, EvaluationContext evalContext, List<Obj
     int indent;
 
     private void printStr (String s) {
+        StringBuilder padding = new StringBuilder();
         for (int i = 0; i < 2 * indent; i++)
-            Std.out.print(" ");
-        Std.out.println(s);
+            padding.append(" ");
```

Comment:
Here we lack Python’s `' ' * 2 * indent`.

Reply:
I'm sure there is a more elegant way to do left padding...